onnxruntime/python/tools/tensorrt/perf/benchmark.py

```diff
@@ -1099,7 +1104,7 @@ def run_onnxruntime(args, models):
                         "fp16": fp16,
                         "io_binding": args.io_binding,
                         "graph_optimizations": args.graph_enablement,
-                        "enable_cache": args.enable_cache,
+                        "enable_cache": eval_bool_str(args.trt_ep_options.get(enable_cache_key)),
```

Comment:
Because it is just a log, we don't need to have the eval_bool_str method but can output the direct value